### PR TITLE
test obt github annotations

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

File: test/BaseTable.test.js [hunk 1]
```diff
@@ -346,7 +346,7 @@ describe("BaseTable", () => {
 
 		it('errors if an invalid column index is provided', () => {
 			proclaim.throws(() => {
-				table.sortRowsByColumn('not a column index', 'descending');
+// 				table.sortRowsByColumn('not a column index', 'descending');
```
Comment (JakeChampion) [inline test/BaseTable.test.js:349]:
```suggestion
 				table.sortRowsByColumn('not a column index', 'descending');
```

File: test/BaseTable.test.js [hunk 2]
```diff
@@ -358,7 +358,7 @@ describe("BaseTable", () => {
 
 		it('errors if an invalid sort order is provided', () => {
 			proclaim.throws(() => {
-				table.sortRowsByColumn(1, 'not a sort order');
+// 				table.sortRowsByColumn(1, 'not a sort order');
```
Comment (JakeChampion) [inline test/BaseTable.test.js:361]:
```suggestion
 				table.sortRowsByColumn(1, 'not a sort order');
```

File: test/BaseTable.test.js [hunk 3]
```diff
@@ -346,7 +346,7 @@ describe("BaseTable", () => {
 
 		it('errors if an invalid column index is provided', () => {
 			proclaim.throws(() => {
-				table.sortRowsByColumn('not a column index', 'descending');
+ 				table.sortRowsByColumn('not a column index', 'descending');
```
Comment (JakeChampion) [inline test/BaseTable.test.js:349]:
```suggestion
				table.sortRowsByColumn('not a column index', 'descending');
```

File: test/BaseTable.test.js [hunk 4]
```diff
@@ -358,7 +358,7 @@ describe("BaseTable", () => {
 
 		it('errors if an invalid sort order is provided', () => {
 			proclaim.throws(() => {
-				table.sortRowsByColumn(1, 'not a sort order');
+ 				table.sortRowsByColumn(1, 'not a sort order');
```
Comment (JakeChampion) [inline test/BaseTable.test.js:361]:
```suggestion
				table.sortRowsByColumn(1, 'not a sort order');
```

**Comment by chee** [comment]
this is really cool:)

**Pull request closed without merging**